201602_002.md

```diff
@@ -0,0 +1,141 @@
+# 201602 002
+
+A couple of days after the first logged event, Fernando, Miguel and Pedro are in a Ruby meetup. This time there were live demos.
```

Comment:
Don't spoil the live demos part, the log already starts with it :stuck_out_tongue: